docs/locale/fr/LC_MESSAGES/userguide/troubleshooting.po

```diff
@@ -16,133 +16,138 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
 "Generated-By: Babel 2.15.0\n"
-"X-Generator: Poedit 3.4.4\n"
+"X-Generator: Poedit 3.5\n"
 
 #: ../../userguide/troubleshooting.rst:6
 msgid "Troubleshooting"
 msgstr "Résolution des problèmes"
 
 #: ../../userguide/troubleshooting.rst:10
-msgid "MS Excel/Word or another MS Office software says the file name is too long"
+msgid ""
+"MS Excel/Word or another MS Office software says the file name is too long"
 msgstr ""
-"MS Excel/Word ou un autre logiciel MS Office indique que le nom du fichier est "
-"trop long"
+"MS Excel/Word ou un autre logiciel MS Office indique que le nom du fichier "
+"est trop long"
 
 #: ../../userguide/troubleshooting.rst:14
 msgid ""
-"This error message occurs when you save or open a file if the path to the file "
-"(including the file name) exceeds 218 characters. This limitation includes "
-"three characters representing the drive, the characters in folder names, the "
-"backslash character between folders, and the characters in the file name."
+"This error message occurs when you save or open a file if the path to the "
+"file (including the file name) exceeds 218 characters. This limitation "
```

Comment:
260 characters including the drive, meaning 256 for the path. (3 characters for the drive D:\ and one for a NULL character).

Reply:
[`8af7270`](https://github.com/Scille/parsec-cloud/pull/8387/commits/8af72704163b00dc0eb1006ad9ed89e521f1731f)